spec/services/stats/r103_sar_business_unit_performace_report_spec.rb

```diff
@@ -1,9 +1,62 @@
 require 'rails_helper'
 
+# rubocop:disable Metrics/ModuleLength
 module Stats
   describe R103SarBusinessUnitPerformanceReport do
 
-    before(:all) { create_report_type(abbr: :r103) }
+    before(:all) { 
+      create_report_type(abbr: :r103)
+
+      @bizgrp_ab = create :business_group, name: 'BGAB'
+      @dir_a     = create :directorate, name: 'DRA', business_group: @bizgrp_ab
+      @dir_b     = create :directorate, name: 'DRB', business_group: @bizgrp_ab
+      @bizgrp_cd = create :business_group, name: 'BGCD'
+      @dir_cd    = create :directorate, name: 'DRCD', business_group: @bizgrp_cd
+
+      @team_a = create :business_unit, name: 'RTA', directorate: @dir_a
+      @team_b = create :business_unit, name: 'RTB', directorate: @dir_b
+      @team_c = create :business_unit, name: 'RTC', directorate: @dir_cd
+      @team_d = create :business_unit, name: 'RTD', directorate: @dir_cd
+      @team_dacu_disclosure = find_or_create :team_dacu_disclosure
+
+      @responder_a = create :responder, responding_teams: [@team_a]
+      @responder_b = create :responder, responding_teams: [@team_b]
+      @responder_c = create :responder, responding_teams: [@team_c]
+      @responder_d = create :responder, responding_teams: [@team_d]
+
+      @sar_tmm = create :refusal_reason, :sar_tmm
+      @vex = create :refusal_reason, :vex
+
+      # create cases based on today's date of 30/6/2017
```

Comment:
What does 'today's date' mean in this context - it was today's date three years ago but not today 🤔

Reply:
@willmcb,  today here is same meaning as the above I explain, normally in all those tests related report, time is quite sensitive and important factor for the outcome (statistic figures), in order to avoid uncertain, we normally will use absolute timing like using Time.freeze, not sue relative in order to get rid of which day is weekend, or bank holiday or Feb thing),  so `today` here means the date the time is freeze at